e2e/explorer/workflow/active.spec.ts

```diff
@@ -1 +1,138 @@
-// add tests for /namespace/explorer/workflow/active tab here.
+import { Page, expect, test } from "@playwright/test";
+import {
+  createNamespace,
+  deleteNamespace,
+} from "../../utils/namespace";
+
+import {
+  createWorkflow,
+} from "../../utils/node";
+import { faker } from "@faker-js/faker";
+
+let namespace = "";
+let workflow = "";
+const defaultDescription = 'A simple \'no-op\' state that returns \'Hello world!\''
+
+test.beforeEach(async ({page}) => {
+  namespace = await createNamespace();
+  workflow = await createWorkflow(namespace, faker.git.shortSha() + '.yaml');
+  test.setTimeout(120000)
+  await page.goto("/");
+  
+  await expect(
+    page.getByTestId("breadcrumb-namespace"),
+    "it renders the breadcrumb for a namespace"
+  ).toBeVisible();
+
+  // at this point, any namespace may be loaded.
+  // let's navigate to the test's namespace via breadcrumbs.
+
+  await page.getByTestId("dropdown-trg-namespace").click();
+  await page
+    .getByRole("option", {
+      name: namespace,
+    })
+    .click();
+
+  await expect(page, "the namespace is reflected in the url").toHaveURL(
+    `/${namespace}/explorer/tree`
+  );
+  await expect(
+    page.getByTestId("breadcrumb-namespace"),
+    "the namespace is reflected in the breadcrumbs"
+  ).toHaveText(namespace);
+
+  await page.getByTestId(`explorer-item-link-${workflow}`).click()
```

Comment:
After navigating to the revisions page, it would be good to check that the breadcrumb for the workflow exists.